### ISSUE TITLE
LDAP login does not work on Invitation email

### ISSUE THREAD

**Issue by flyingcircle** [ensky/taiga-contrib-ldap-auth#18]
When sending an invite to new members, the invite email login does not use the ldap login process.

**Comment by artlepool** [comment]
Hey @flyingcircle 

Firstly, curious to know why you need to send out invites when Taiga is configured to use ldap, as anybody that has an entry in your ldap server can log in without the need to be invited?
I don't see anything particularly wrong with this, but at my company we don't send invites out, we send a link to Taiga and anybody can log in if they want to, as their user accounts are already in Active Directory.

Also, it is Taiga itself (rather than this plugin) that builds and sends out the invites so I think this would probably need to be logged as an issue against the main Taiga project.  All this plugin does is handle the login process when the Taiga app asks it to...

**Comment by flyingcircle** [comment]
Well the issue was if I wanted to add someone to a project that has not signed into Taiga. They are sent an invitation email that they are supposed to ignore because the login process doesn't work.

I recently went through saying everyone needed to login first to avoid this confusion. I figured that since the invitation was considered a "private" login it does not go through the same process.

**Comment by mroe1234** [comment]
@flyingcircle I am having the same issue.  user 1 created a project and has invited user 2, 3, and 4 to join it.  They can't 'accept' the invitation because the link doesn't allow them to login.  Can you clarify how you adjusted your workflow to avoid this bug?

**Comment by mroe1234** [comment]
Looks like I can send them an invitation with instructions saying "go to "servername" and log in" then email me when it is done, so that I can then just add them to the project.  Kind of a hack but it'll work.

**Comment by flyingcircle** [comment]
Yes that is what I ended up having to do. I would also warn you that people should not use the change password form in the user account settings. If they do, I have found no way to clear it or prevent users from using the change password form.

On Sep 16, 2015 6:40 PM, mroe1234 notifications@github.com wrote:
Looks like I can send them an invitation with instructions saying "go to <servername> and log in" then email me when it is done, so that I can then just add them to the project.  Kind of a hack but it'll work.

---

**Comment by mroe1234** [comment]
Looking at the code the best I can, it looks like the invitation login doesn't respect the loginFormType variable.

**Comment by mgrobelin** [comment]
Invite mail doesn't work for me either while having LDAP auth enabled. One can login, but is not attached to the corresponding project. What a bummer, as this currently negates the use of having this kind of auth 😞

**Comment by muppeth** [comment]
I did some poking around. Tried changing invitation template but taiga still uses default login/register form.
What I noticed is that if user registers on the invitation form with ldap uid they will be assigned to the project. Next time they login with ldap credentials and it works. In registration form they don't even have to put real password as this gets overwritten by ldap after next login. 

What I ended up doing is to modify the template and remove login form, leaving only registration. I put a note that the username must be the same as LDAP username.

I know it is a rather dirty hack, but for the time being it works.
